validator/sawtooth_validator/execution/scheduler.py

```diff
@@ -187,6 +187,17 @@ def get_transaction(self, index):
         """
         raise NotImplementedError()
 
+    def get_batch_id_for_transaction(self, transaction_id):
```

Comment:
I think that we might want to make the caller keep a record of the batches/transactions it added to the scheduler. Eventually this API may be across a 0MQ socket.  I'll approve either way, but something to consider I think.